samples/wifi/raw_tx_packet/src/wifi_connection.c

```diff
@@ -196,7 +196,7 @@ static int __wifi_args_to_params(struct wifi_connect_req_params *params)
 static int wifi_connect(void)
 {
 	struct net_if *iface = NULL;
-	static struct wifi_connect_req_params cnx_params;
+	static struct wifi_connect_req_params cnx_params = { 0 };
```

Comment:
Why is `static` in use in the first place? I don't see any necessity.

Reply:
And just initialising zero doesn't mean that the values are sane, you need to have a `set_defaults` which you did in https://github.com/nrfconnect/sdk-nrf/pull/13715

Reply:
> Why is `static` in use in the first place? I don't see any necessity.

Will remove and raise new PR.